frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/BuilderAndDesignDrawer.tsx

```diff
@@ -25,6 +25,9 @@ export const BuilderAndDesignDrawer = (): JSX.Element | null => {
   const { activeTab, isDrawerOpen } = useCreatePageSidebar()
   const createOrEditData = useFieldBuilderStore(stateDataSelector)
 
+  const { useMeasureDrawer } = useCreatePageSidebar()
+  const [drawerRef] = useMeasureDrawer
+
```

Comment:
This can probably be merged into line 25 instead:

```typescript
const { activeTab, isDrawerOpen, useMeasureDrawer: [drawerRef] } = useCreatePageSidebar()
```

Reply:
tunnel vision got me, fixed. kept it as 2 lines as its a tad more readable imo